FreeRTOS-Plus/Source/Utilities/logging/logging_stack.h

```diff
@@ -37,25 +37,58 @@
 #include "logging_levels.h"
 
 /* Standard Include. */
+#include <stdio.h>
 #include <stdint.h>
 
+/* The macro definition for LIBRARY_LOG_NAME is for Doxygen
```

Comment:
I don't understand this comment as it implies the macro is only used by Doxygen, but the comment below says the macro is also used in log messages.

Reply:
This comment was a copy-paste from the [C-SDK version of logging_stack.h](https://github.com/aws/aws-iot-device-sdk-embedded-C/blob/master/demos/logging-stack/logging_stack.h#L39-L60) which contains special text for doxygen generation. I have removed this comment about doxygen generation as it is not relevant in this repository.